infra/modules/cicd/main.tf

```diff
@@ -80,43 +116,134 @@ resource "google_cloudbuild_trigger" "app_new_build" {
       invert_regex = false
```

Comment:
This won't let me highlight it but this is referring to `line 115`. Switch regex branch from just `main` to `^main$`.

Reply:
done